site/docs/guide.md

```diff
@@ -1078,146 +1078,4 @@ these messages.
-Total execution phase time              49.473 s
-Total time finalizing build              83.9 ms
-
-Action dependency map creation           0.00 ms
-Actual execution time                   49.473 s
-
-Total time (across all threads) spent on:
-              Type    Total    Count     Average
-            ACTION    2.25%    12229     10.2 ms
-[...]
-       SKYFUNCTION    1.87%   236131     0.44 ms
-```
-
-The last section shows the critical path:
-
-```
-Critical path (32.078 s):
-    Id        Time Percentage   Description
-1109746     5.171 s   16.12%   Building [...]
-1109745      164 ms    0.51%   Extracting interface [...]
-1109744     4.615 s   14.39%   Building [...]
-[...]
-1109639     2.202 s    6.86%   Executing genrule [...]
-1109637     2.00 ms    0.01%   Symlinking [...]
-1109636      163 ms    0.51%   Executing genrule [...]
-           4.00 ms    0.01%   [3 middleman actions]
-```
-
-You can use the following options to display more detailed information:
-
--   <a id="dump-text-format"></a>[`--dump=text`](user-manual.html#flag--dump)
-
-    This option prints all recorded tasks in the order they occurred. Nested
-    tasks are indented relative to the parent. For each task, output includes
-    the following information:
-
-    ```
-    [task type] [task description]
-    Thread: [thread id]    Id: [task id]     Parent: [parent task id or 0 for top-level tasks]
-    Start time: [time elapsed from the profiling session start]       Duration: [task duration]
-    [aggregated statistic for nested tasks, including count and total duration for each nested task]
-    ```
-
--   <a id="dump-raw-format"></a>[`--dump=raw`](user-manual.html#flag--dump)
-
-    This option is most useful for automated analysis with scripts. It outputs
-    each task record on a single line using '|' delimiter between fields. Fields
-    are printed in the following order:
-
-    1.  thread id - integer positive number, identifies owner thread for the
-        task
-    2.  task id - integer positive number, identifies specific task
-    3.  parent task id for nested tasks or 0 for root tasks
-    4.  task start time in ns, relative to the start of the profiling session
-    5.  task duration in ns. Please note that this will include duration of all
-        subtasks.
-    6.  aggregated statistic for immediate subtasks per type. This will include
-        type name (lower case), number of subtasks for that type and their
-        cumulative duration. Types are space-delimited and information for
-        single type is comma-delimited.
-    7.  task type (upper case)
-    8.  task description
-
-    Example:
-
-    ```
-    1|1|0|0|0||PHASE|Launch Bazel
-    1|2|0|6000000|0||PHASE|Initialize command
-    1|3|0|168963053|278111411||VFS_READLINK|/[...]
-    1|4|0|571055781|23495512||VFS_STAT|/[...]
-    1|5|0|869955040|0||PHASE|Load packages
-    [...]
-    ```
-
-If Bazel appears to be hung, you can hit <kbd>Ctrl-&#92;</kbd> or send
```

Comment:
This and the following lines need to be kept (or moved to the referenced place).

Reply:
Done. Moved it; couldn't easily fit it into the profiling flow of text, so made a new subsection.